code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -441,7 +441,8 @@
 	G.mind.store_memory("<b>Serve [user.real_name], your creator.</b>")
 	if(user.mind.special_role)
 		message_admins("[G.real_name] has been summoned by [user.real_name], an antagonist.")
```

Comment:
Question, why does the game tell the admins that an antag created a golem...but the golem is not told of its master being an antagonist?